### PR TITLE
APIGOV-23841 - add trace logging for flush out harvester connect issue

### PR DESCRIPTION
**Pull request by sbolosan**
@jcollins-axway @tsjohns9 - I feel like some of the trace logging is lame, like doesn't really do anything except leave bread crumbs to the real error.  And maybe that's what we want.  Let me know if you need me to add more/take away ...  thanks.

### CODE REVIEW

**Review by jcollins-axway — CHANGES_REQUESTED** [review]
This does not need to be merged into the SDK, until we discover the actual issue. We can give a feature build for just the logging while we investigate.

**Comment by jcollins-axway** [comment]
I am fine with all the logging, verbose or not, as it is trace level and meant only for developers

**Comment by sbolosan** [comment]
@jcollins-axway right.. i will close this after we get what we need.  I already pinged Ethan and that's where I was going... was going to ask if he is ok taking a build from me

File: pkg/agent/poller/client.go [hunk 1]
```diff
@@ -63,16 +64,22 @@ func NewPollClient(
 
 // Start the polling client
 func (p *PollClient) Start() error {
+	logger := log.NewFieldLogger().
+		WithComponent("pollExecutor").
```
Comment (tsjohns9) [inline pkg/agent/poller/client.go:68]: should be PollClient

**Review by tsjohns9 — DISMISSED** [review]

File: pkg/agent/poller/client.go (hunk 1 above)
Comment (sbolosan) [reply pkg/agent/poller/client.go:68]: oops, you're right... copy paste

**Review by tsjohns9 — DISMISSED** [review]

**Comment by sbolosan** [comment]
Reverted most of the traces.  Now on startup, using ticker and ticker timeout to check hc status.
ticker - check every 5 seconds
tickerTImeout - exhaust after 5 minutes...

fail if 5 mins exhaust

**Comment by jcollins-axway** [comment]
I think some of the logging that was added may be beneficial. It may be verbose but in some cases nice to see when the code gets in and out of functions.

**Review by jcollins-axway — DISMISSED** [review]

**Comment by sbolosan** [comment]
@jcollins-axway , i have the trace version sent out to Ethan/laura to test tomorrow.  I will reintroduce the ones that make sense 🤙🏾

**Review by jcollins-axway — APPROVED** [review]

**Review by vivekschauhan — APPROVED** [review]

**Pull request merged**